libraries/chain/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ if("eos-vm-oc" IN_LIST EOSIO_WASM_RUNTIMES)
 
    set_source_files_properties(webassembly/runtimes/eos-vm-oc/LLVMJIT.cpp PROPERTIES COMPILE_FLAGS "--std=gnu++17")
    set_source_files_properties(webassembly/runtimes/eos-vm-oc/LLVMEmitIR.cpp PROPERTIES COMPILE_FLAGS "--std=gnu++17")
+   set_source_files_properties(webassembly/runtimes/eos-vm-oc/switch_stack_linux.s PROPERTIES COMPILE_FLAGS "-Wno-unused-command-line-argument")
```

Comment:
I think this is a fine fix.. but I wonder why cmake is operating this way :thinking:  it clearly knows that `.s` are assembly files -- not c/c++ files -- so why is it tacking on all the c/c++ options like `-O3 -DNDEBUG` etc

Reply:
beats me!